README.md

```diff
@@ -4,9 +4,13 @@
 [![Go Report Card](https://goreportcard.com/badge/prometheus-operator/prometheus-operator "Go Report Card")](https://goreportcard.com/report/prometheus-operator/prometheus-operator)
 [![Slack](https://img.shields.io/badge/join%20slack-%23prometheus--operator-brightgreen.svg)](http://slack.k8s.io/)
 
-**Project status: *beta*** Not all planned features are completed. The API, spec, status and other user facing objects may change, but in a backward compatible way.
+## Project Status
```

Comment:
This is currently top level, I would probably put it under the Overview, let me know what you think.

Reply:
I'm fine with moving it! If by chance we notice that users are not finding this info we can always move it back here

Reply:
moved it 👍